apps/clinical/src/pages/__tests__/ConsultationPage.test.tsx

```
@@ -542,4 +542,144 @@ describe('ConsultationPage', () => {
+      expect(
+        screen.getByTestId('loading-dashboard-config-test-id'),
+      ).toBeInTheDocument();
+    });
+
+    it('should render dashboard container when config loads successfully', async () => {
+      renderWithProvider();
+
+      await waitFor(() => {
+        expect(
+          screen.getByTestId('dashboard-container'),
+        ).toBeInTheDocument();
```

ESLint

[error] Replace `⏎··········screen.getByTestId('dashboard-container'),⏎········` with `screen.getByTestId('dashboard-container')`

(prettier/prettier)